kurl_proxy/cmd/main.go

```diff
@@ -621,14 +620,17 @@ func generateDefaultCertSecret(secrets corev1.SecretInterface, namespace string)
 			Name:      "kotsadm-tls",
 			Namespace: namespace,
 			Annotations: map[string]string{
-				"acceptAnonymousUploads": "0",
+				"acceptAnonymousUploads": "1",
 			},
 		},
 		Type:       "kubernetes.io/tls",
 		Data:       make(map[string][]byte),
 		StringData: make(map[string]string),
 	}
 
+	// TODO: Why is the namespace always "default"?
+	// Requests to kotsadm.embeddded-cluster.svc.cluster for example will
+	// fail with invalid cert errors
```

Comment:
Not sure about the historical reasons, but kots and kurl-proxy are always deployed to the default namespace with the kurl add-on. If it needs to be other namespaces, i think we could make it configurable. Maybe even use the `namespace` variable we already have in the main function that comes from [here](https://github.com/replicatedhq/kots/blob/main/deploy/kurl/kotsadm/template/base/kurl-proxy/tmpl-deployment.yaml#L33-L36)?

Reply:
I left the hostname as-is, but I have added DNS alt names to the cert in case the namespace is not default. This will keep any old assumptions valid such as in [ekco's rotation logic](https://github.com/replicatedhq/ekco/blob/f879a259ecff9e25ac7a3685561aa39e2ec3b61a/pkg/cluster/kotsadm.go#L57)